src/components/accessories/splitButton/SplitButton.tsx

```diff
@@ -0,0 +1,104 @@
+import MenuList from '@material-ui/core/MenuList';
+import { IProps } from "./types";
+import { useTranslation } from "react-i18next";
+
+const SplitButton: FunctionComponent<IProps> = ({
+  type,
+  label,
+  descriptions,
+  className,
+  onClick
+}) => {
+  const [open, setOpen] = React.useState(false);
+  const anchorRef = React.useRef<HTMLDivElement>(null);
+  const [selectedIndex, setSelectedIndex] = React.useState(1);
+
+  const { t } = useTranslation();
+
+  // const handleClick = () => {
+  //   console.info(`You clicked ${descriptions[selectedIndex]}`);
+  // };
+
+  // const handleMenuItemClick = (
+  //   event: React.MouseEvent<Element, MouseEvent>,
+  //   index: number,
+  // ) => {
+  //   onClick(index);
+  // };
+
+  const handleToggle = () => {
+    setOpen((prevOpen) => !prevOpen);
+  };
+
+  const handleClose = (event: React.MouseEvent<Document, MouseEvent>) => {
+    if (anchorRef.current && anchorRef.current.contains(event.target as HTMLElement)) {
+      return;
+    }
+
+    setOpen(false);
+  };
+
+  return (
+    <span ref={anchorRef} >
+      {/*   <ButtonGroup  aria-label="split button"> color="primary" ref={anchorRef} */}
+          {/* <Button
+            onClick={handleClick}>{label}
+          </Button> */}
+          <Button
+            type={type}
+            // variant="contained"
+            // color="primary"
+            // size="small"
+            className={className}
+            aria-controls={open ? 'split-button-menu' : undefined}
+            aria-expanded={open ? 'true' : undefined}
+            // aria-label={label}
+            aria-haspopup="menu"
+            onClick={handleToggle}
+          > 
+          {label}
+            {/* <div className="button__label"> {label}</div> */}
+            {/* <ArrowDropDownIcon /> */}
+          </Button>
+        {/* </ButtonGroup> */}
+        <Popper open={open} anchorEl={anchorRef.current} role={undefined} transition>
+          {({ TransitionProps, placement }) => (
+            <Grow
+              {...TransitionProps}
+              style={{
+                transformOrigin: placement === 'bottom' ? 'center top' : 'center bottom',
+              }}
+            >
+              <Paper>
+                <ClickAwayListener onClickAway={handleClose}>
+                  <MenuList id="split-button-menu">
+                    { descriptions.map((description, index) =>
+                     (
+                        <MenuItem
+                          key={index}
+                          onClick={(event) => { if(onClick) onClick(event, index); handleToggle(); }}
+                          >
+                          {description}
+                        </MenuItem>)
+                    )
+                    }
+                  </MenuList>
+                </ClickAwayListener>
+              </Paper>
+            </Grow>
+          )}
+        </Popper>
+        </span>
+  );
+}
+
+export default SplitButton;
```

Comment:
We're not gonna usa submenu inside the main menu in due to the fact that each section should be trated as a landing page. The subsections should be visibile only once you went on that page. 

If this component is not gonna use, remove it.